### PR TITLE
Remove pullback for `Wishart`

### PR DESCRIPTION
**Pull request by devmotion**
This should be fixed by https://github.com/JuliaStats/Distributions.jl/pull/1410.

### CODE REVIEW

File: src/zygote.jl [hunk 1]
```diff
@@ -35,30 +35,6 @@ end
...
-        prom_S = convert(AbstractArray{T}, S)
-        Wishart{R, typeof(prom_S), typeof(rnk)}(R(df), prom_S, R(logc0), rnk, singular)
-    end
-    return ZygoteRules.pullback(_Wishart, df, S, warn)
-end
-
-_warn(msg) = @warn(msg)
-@non_differentiable _warn(msg)
-
 ZygoteRules.@adjoint function Distributions.Wishart(df::Real, S::AbstractMatrix{<:Real})
```
Comment (yebai) [inline src/zygote.jl:38]: Do we still need to keep this `adjoined?

**Review by yebai — APPROVED** [review]
Thanks, @devmotion - it looks good other than the minor comment below.

File: src/zygote.jl (hunk 1 above)
Comment (devmotion) [reply src/zygote.jl:38]: I wasn't sure and hence planned to check it separately. I guess its motivation is that it avoids the construction of a `PDMat` and instead works with the Cholesky factors directly (https://github.com/TuringLang/DistributionsAD.jl/blob/9461c6b9eb3472efce160d37fbe1a1b4ec8c9bb6/src/matrixvariate.jl#L7-L11).

Comment (devmotion) [reply src/zygote.jl:38]: Is it fine with you if we merge this PR and investigate the question later, together with other PDMats workarounds @yebai?

Comment (yebai) [reply src/zygote.jl:38]: No problem - happy to merge this PR as-is!

**Pull request merged**